orcid-pub-web/src/main/java/org/orcid/api/filters/ApiRateLimitFilter.java

```diff
@@ -297,5 +302,18 @@ private boolean isWhiteListed(String ipAddress) {
         }
         return false;
     }
+    
+    private boolean isClientIdWhiteListed(String clientId) {
+        List<String> papiClientIdWhiteList = null;
+        if (StringUtils.isNotBlank(papiClientIdWhiteSpaceSeparatedWhiteList)) {
```

Comment:
Lets change this so the list is generated on constructor time and here we only need to check if the id is in the list